google/cloud/firestore_v1/document.py

```diff
@@ -402,16 +403,26 @@ def get(
             create_time = None
             update_time = None
         else:
-            data = _helpers.decode_dict(document_pb.fields, self._client)
-            exists = True
-            create_time = document_pb.create_time
-            update_time = document_pb.update_time
+            batched_response: BatchGetDocumentsResponse = list(batch_response_pb)[0]
```

Comment:
while this is not likely a problem, this could result in oob.

Reply:
What this is really highlighting is that that `NotFound` exception is no longer relevant, since missed documents still return empty responses. So, as far as I can tell, using the `BatchGetDocuments` proto, this list will always have an element.

But, all that said, I think I'll both remove the `NotFound` exception and explicitly check the length of this list.

Reply:
Truthiness check on `response` now avoids an empty list.